apps/etterlatte-saksbehandling-ui/client/src/shared/api/feature.ts

```diff
@@ -1,6 +1,16 @@
 import { apiClient, ApiResponse } from '~shared/api/apiClient'
 import { IFeature } from '~shared/types/IFeature'
+import { logger } from '~utils/logger'
 
 export const hentFunksjonsbrytere = async (brytere: string[]): Promise<ApiResponse<IFeature[]>> => {
-  return apiClient.post(`/feature`, { features: brytere })
+  function post(): Promise<ApiResponse<IFeature[]>> {
+    return apiClient.post(`/feature`, { features: brytere })
+  }
+
+  try {
+    return post()
```

Comment:
Hjelper dette, da? Vet du hvorfor første kallet ikke funker, mens det andre funker?

Reply:
> Hjelper dette, da? Vet du hvorfor første kallet ikke funker, mens det andre funker?

Håper det 🤞 Prøver å løyse at vi iblant får 503-feil frå Unleash, som tilsynelatande fungerer igjen rett etterpå, feks https://nav-it.slack.com/archives/C043C5ZGW5T/p1711107925812949